src/uct/base/uct_md.c

```diff
@@ -465,6 +465,19 @@ uct_md_mem_advise(uct_md_h md, uct_mem_h memh, void *addr, size_t length,
 ucs_status_t uct_md_mem_reg(uct_md_h md, void *address, size_t length,
                             unsigned flags, uct_mem_h *memh_p)
 {
+    uct_md_mem_reg_params_t params = {
+        .field_mask = UCT_MD_MEM_REG_FIELD_FLAGS,
+        .flags      = flags
+    };
+
+    return uct_md_mem_reg_v2(md, address, length, &params, memh_p);
+}
+
+ucs_status_t uct_md_mem_reg_v2(uct_md_h md, void *address, size_t length,
+                               const uct_md_mem_reg_params_t *params,
+                               uct_mem_h *memh_p)
+{
+    uint64_t flags = UCT_MD_MEM_REG_FIELD_VALUE(params, flags, FIELD_FLAGS, 0);
```

Comment:
should be unsigned (or convert flags to uint64_t)

Reply:
converted to `uint64_t`